spec/features/courses/fees_spec.rb

```diff
@@ -64,6 +64,28 @@
     expect(current_path).to eq description_provider_course_path('AO', course_1.course_code)
   end
 
+  context 'with course_length_other selected' do
+    let(:course_1) do
+      jsonapi(
+        :course,
+        :with_fees,
+        course_length: '6months',
```

Comment:
```suggestion
        course_length: '6 months',
```

closer to real life example